### ISSUE TITLE
90% of Russian texts are displayed in Times New Roman

### ISSUE THREAD

**Issue by DarkTraveler** [strapi/strapi#6643]
**Describe the bug**
Hi. In the new version of strapi 3.0 / 3.0.1 support for cyrillic characters has disappeared.

90% of Russian texts are displayed in Times New Roman.
This problem is also present when installing strapi from scratch.

Beta: 
![image](https://user-images.githubusercontent.com/16899697/84468305-d88ce780-acc1-11ea-8b2d-7d87ef03b896.png)

3.0.1
![image](https://user-images.githubusercontent.com/16899697/84468359-f65a4c80-acc1-11ea-914f-4c49ae372785.png)

**System**
Node.js version: 12.14.1
NPM version: 6.14.5
Strapi version: 3.0.1
Database: mongodb
Operating system: ubuntu 20

**Comment by lauriejim** [comment]
I ping @soupette and @Aurelsicoko about this topic.

**Comment by MoreBlood** [comment]
Not only font are broken, but also some buttons messed up with chrome default apperance
![strapi-issue-1](https://user-images.githubusercontent.com/10070770/84891720-f9c84c00-b0a4-11ea-9299-4d0eaa9876c7.JPG)

The issue is that font **Lato** don't have cyrilic font face, and for some elements it's overwritten without fallback
![strapi-issue-2](https://user-images.githubusercontent.com/10070770/84891721-fa60e280-b0a4-11ea-860b-b3495d1bef1b.JPG)

I think it's buffetjs issue, font used: [Lato on google fonts](https://fonts.google.com/specimen/Lato?preview.text=+something+on+cyrillic+-+%D1%87%D1%82%D0%BE-%D1%82%D0%BE+%D0%BD%D0%B0+%D0%BA%D0%B8%D1%80%D0%B8%D0%BB%D0%BB%D0%B8%D1%86%D0%B5&preview.text_type=custom#standard-styles)

**Comment by MoreBlood** [comment]
As temporary workaround, include this in some modified file of admin panel

```css
button,
input,
table,
td,
tr {
  background: none;
  border: 0;
  border-collapse: collapse;
  border-image-width: 0;
  border-spacing: 0;
  color: inherit;
}

body * {
  font-family: Lato, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial,
    'Noto Sans', sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol',
    'Noto Color Emoji' !important;
}

.count-info {
  white-space: nowrap;
}
```

**Comment by Ashot-KR** [comment]
It's because of Lato from google fonts does not contains cyrillic chars.

**Comment by Ashot-KR** [comment]
I was wrong – fonts comes from buffetjs, there is same problem with cyrillic characters 
![image](https://user-images.githubusercontent.com/899218/86406664-365a9100-bcbc-11ea-82c7-147459089bbe.png)

**Comment by Ashot-KR** [comment]
made [PR to buffet](https://github.com/strapi/buffet/pull/140) to fix issue with fonts

**Comment by Ashot-KR** [comment]
issue fixed (tested on strapi-admin@3.1.3)
![image](https://user-images.githubusercontent.com/899218/89231216-8bf0c900-d5ed-11ea-915b-ab5b92b878ae.png)

**Comment by soupette** [comment]
@Ashot-KR I am closing the issue then
